daemons/dss-sync-sfn/app.py

```diff
@@ -49,7 +49,9 @@ def launch_from_s3_event(event, context):
                 if exists(dest_replica, obj.key):
                     # Logging error here causes daemons/invoke_lambda.sh to report failure, for some reason
                     # - Brian Hannafious, 2019-01-31
-                    logger.info("Key %s already exists in %s, skipping sync", obj.key, dest_replica)
+                    logger.info("[s3 Event] Key %s already exists in %s, skipping sync.",
```

Comment:
I think the logger converts objects to strings implicitly, e.g.
```
In [16]: class Foo: 
    ...:     pass 
    ...:                                                                                                                                                                             

In [17]: f = Foo()                                                                                                                                                                   

In [18]: logging.error("my message: %s", f)                                                                                                                                          
ERROR:root:my message: <__main__.Foo object at 0x7f634c4c1128>

In [19]: str(f)                                                                                                                                                                      
Out[19]: '<__main__.Foo object at 0x7f634c4c1128>'
```

Reply:
Ah, my mistake.